src/snowflake/snowpark/modin/plugin/docstrings/series.py

```diff
@@ -3792,6 +3792,7 @@ def nunique():
 
         Examples
         --------
+        >>> import snowflake.snowpark.modin.pandas as pd
```

Comment:
This should already been imported here snowflake/snowpark/modin/conftest.py

Reply:
Reverted, I added this line by mistake.